stripe-test-e2e/src/test/java/com/stripe/android/test/e2e/EndToEndTest.kt

```diff
@@ -261,7 +261,7 @@ internal class EndToEndTest {
     }
 
     @Test
-    fun `test us_bank_account payment intent flow with desciptor code`() = runTest {
+    fun `test us_bank_account payment intent flow with descriptor code`() = runTest {
```

Comment:
I think it may be worthwhile to add end to end tests for these new payment methods